tst/testinstall/recordname.tst

```diff
@@ -28,6 +36,28 @@ gap> r.(str1023) := 4;
 Error, Record names must consist of less than 1023 characters
 gap> r.(str1024) := 5;
 Error, Record names must consist of less than 1023 characters
+gap> names := RecNames(r);;
+gap> SortedList(names) = [str1021, str1022a, str1022b, "x", "y"];
+true
+gap> RecNames( () );
+Error, no method found! For debugging hints type ?Recovery from NoMethodFound
+Error, no 1st choice method found for `RecNames' on 1 arguments
+gap> REC_NAMES( () );
+Error, RecNames: <rec> must be a record (not a permutation (small))
+gap> REC_NAMES_COMOBJ( () );
+Error, RecNames: <rec> must be a component object (not a permutation (small))
+gap> EQ_PREC( rec(), () );
+false
+gap> EQ_PREC( (), rec() );
+false
+gap> LT_PREC( rec(), () );
+false
+gap> LT_PREC( (), rec() );
+true
```

Comment:
These tests seem oddly restricted. I.e. only "bad" comparison are tested, but there are no comparisons of records with other records. Again: Of course this is no reason for blocking this PR, I am just wondering